### ISSUE TITLE
DeleteManyAsync problem with Oracle and Elsa.Persistence.EntityFrameworkCore

### ISSUE THREAD

**Issue by borjaferv** [elsa-workflows/elsa-core#3456]
Hey, recently i upgrade to elsa 2.9.2 and cant make it work with my oracle 21. I run migrations manually (all fine) , but the DeleteManyAsync is failing:

``` System.InvalidCastException: Unable to cast object of type 'Microsoft.Data.SqlClient.SqlParameter' to type 'Oracle.ManagedDataAccess.Client.OracleParameter'.
   at Oracle.EntityFrameworkCore.Storage.Internal.OracleStringTypeMapping.CreateParameter(DbCommand command, String name, Object value, Nullable`1 nullable)
   at Microsoft.EntityFrameworkCore.Storage.Internal.TypeMappedRelationalParameter.AddDbParameter(DbCommand command, Object value)
   at EFCore.BulkExtensions.IQueryableExtensions.ToParametrizedSql(IQueryable query)
   at Elsa.Persistence.EntityFramework.Core.Stores.EntityFrameworkStore`2.<>c__DisplayClass11_0.<<DeleteManyAsync>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at Elsa.Persistence.EntityFramework.Core.Stores.EntityFrameworkStore`2.DoWork[TResult](Func`2 work, CancellationToken cancellationToken)
   at Elsa.Persistence.EntityFramework.Core.Stores.EntityFrameworkStore`2.DoWork[TResult](Func`2 work, CancellationToken cancellationToken)
   at Elsa.Persistence.EntityFramework.Core.Stores.EntityFrameworkStore`2.DeleteManyAsync(ISpecification`1 specification, CancellationToken cancellationToken)
   at Elsa.Services.Triggers.TriggerIndexer.DeleteTriggersAsync(String workflowDefinitionId, CancellationToken cancellationToken)
   at Elsa.Services.Triggers.TriggerIndexer.IndexTriggersAsync(IWorkflowBlueprint workflowBlueprint, CancellationToken cancellationToken)
   at Elsa.Services.Triggers.TriggerIndexer.IndexTriggersAsync(IEnumerable`1 workflowBlueprints, CancellationToken cancellationToken)
   at Elsa.Services.Triggers.TriggerIndexer.IndexTriggersAsync(CancellationToken cancellationToken)
   at Elsa.StartupTasks.IndexTriggers.ExecuteAsync(CancellationToken cancellationToken)
   at Elsa.Runtime.StartupRunner.StartupAsync(CancellationToken cancellationToken)
   at Elsa.Runtime.StartupRunner.StartupAsync(CancellationToken cancellationToken)
   at Elsa.HostedServices.StartupRunnerHostedService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.Run(IHost host) ```


I tried to upgrade to the latest preview 2.10.0-preview-557 but nothing....

**Comment by borjaferv** [comment]
i think is related with:
#3323

**Comment by akunzai** [comment]
Maybe duplicated with #3430, Fixed in #3446

Try downgrade Elsa version from 2.9.2 to 2.9.1

**Comment by borjaferv** [comment]
Nothing...
Fails anyway from 2.9.0 to 2.10.xxxx
Only works at 2.8.2
